src/components/ReadoutGrid.tsx

```diff
@@ -1,15 +1,44 @@
-import React from "react";
+import React, { useState } from "react";
 import "styles/_readout_grid.scss";
-import Inventory from "./Inventory";
+import "styles/_dropdown.scss";
+import "styles/_containers_and_frames.scss";
+import Weapons from "./Weapons";
 import SpellRepertoire from "./SpellRepertoire";
 import InfoBlocks from "./InfoBlocks";
+import PlayerStatsReadout from "./PlayerStatsReadout";
+
+function ReadoutGrid({ monsters, weapons, spells, playerStats }) {
+  const [readout, setReadout] = useState({

```

Comment:
Typescript tip, you can use <> to declare what your type expectation for your `useState` is. So you can do `useState<{ state: string}>({ state: "Weapons" })` (or just `useState<string>("Weapons")`, see above/below)